bank-sdk/sdk/src/main/java/net/gini/android/bank/sdk/capture/skonto/help/SkontoHelpFragment.kt

```diff
@@ -0,0 +1,390 @@
+        AndroidView(factory = {
+            customBottomNavBarAdapter.viewAdapter.onCreateView(FrameLayout(ctx))
+        }, update = {
+            with(customBottomNavBarAdapter.viewAdapter) {
+                setOnBackClickListener(onBackClicked)
+            }
+        })
+    }
+}
+
+@Composable
+private fun HelpFooterSection(
+    colors: SkontoHelpFooterSectionColors,
+    modifier: Modifier = Modifier,
+) {
+    Column(
+        modifier = modifier
+            .fillMaxWidth()
+            .padding(24.dp),
+        horizontalAlignment = Alignment.Start,
+    ) {
+        Text(
+            text = stringResource(id = R.string.gbs_skonto_help_footer_text),
+            style = GiniTheme.typography.body2,
+            color = colors.textColor,
+        )
+
+    }
+}
+
+@Composable
+private fun HelpItemsSection(
+    colors: SkontoHelpItemsSectionColors,
+    modifier: Modifier = Modifier,
+) {
+    Column(
+        modifier = modifier
+            .fillMaxWidth()
+            .background(colors.backgroundColor),
+        //.padding(16.dp),
+        horizontalAlignment = Alignment.CenterHorizontally,
+    ) {
+        HelpItem(
+            modifier = Modifier.padding(16.dp),
+            colors = colors,
+            textId = R.string.gbs_skonto_help_item_capture_invoice,
+            iconId = R.drawable.gbs_skonto_help_capture_invoice
+        )
+        HorizontalDivider(
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(start = 80.dp, end = 16.dp),
+            color = colors.dividerColor,
+            thickness = 1.dp,
+        )
+        HelpItem(
+            modifier = Modifier.padding(16.dp),
+            colors = colors,
+            textId = R.string.gbs_skonto_help_item_identify_discount,
+            iconId = R.drawable.gbs_skonto_help_identify_discount
+        )
+        HorizontalDivider(
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(start = 80.dp, end = 16.dp),
+            color = colors.dividerColor,
+            thickness = 1.dp,
+        )
+        HelpItem(
+            modifier = Modifier.padding(16.dp),
+            colors = colors,
+            textId = R.string.gbs_skonto_help_item_adjust_payment_details,
+            iconId = R.drawable.gbs_skonto_help_adjust_payment_details
+        )
+        HorizontalDivider(
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(start = 80.dp, end = 16.dp),
+            color = colors.dividerColor,
+            thickness = 1.dp,
+        )
+        HelpItem(
+            modifier = Modifier.padding(16.dp),
+            colors = colors,
+            textId = R.string.gbs_skonto_help_item_pay_promptly,
+            iconId = R.drawable.gbs_skonto_help_pay_promptly
+        )
+
+    }
+}
+
+
+@Composable
+private fun HelpItem(
+    colors: SkontoHelpItemsSectionColors,
+    modifier: Modifier = Modifier,
+    textId: Int,
+    iconId: Int,
```

Comment:
[Requirement]
Let's pass here `painter` instead or resId. It will be more flefible

Reply:
Also, this parameter should be placed before Modifier because it's mandatory